catalog/dags/data_refresh/create_filtered_index_dag.py

```diff
@@ -54,9 +54,10 @@
 
 from airflow import DAG
 from airflow.models.param import Param
+from es.create_new_es_index.create_new_es_index_types import CREATE_NEW_INDEX_CONFIGS
```

Comment:
Interesting, I did not notice this. FWIW the imports actually fail with the name `elasticsearch`. Do you have any alternative suggestions?

Reply:
Oh dear 😅 maybe `elasticsearch_cluster`? Feels verbose but might be best to distinguish!

Reply:
Actually weirdly I was able to just move them to the local import block and it seems to work -- not sure how that happened in the first place, or why it didn't _complain_ about it being in the third party block. What do you think? I can still change to `elasticsearch_cluster` if that's preferable!

Reply:
Huh, weird! I guess, if you put it in a 4th, separate block, what does isort/ruff do with them by default? If it's default is to put them in 3rd party rather than local, then I think we should change the name so we don't run into it in the future. If it was just a fluke and it defaults to local instead, that should be fine!

Reply:
Funny development -- I actually just realized that _no_ import sorting is working right now (filed an issue at #3695). Lucky break that this caused me to notice it!

I'm going to assume then that `es` was correctly sorted to the third party block back when this was working, and is not being sorted now because the pre-commit is buggy :) I have gone ahead and made the change to `elasticsearch_cluster` to be totally safe.